contrib/node/src/python/pants/contrib/node/tasks/node_install.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.contrib.node.tasks.node_paths import NodePathsLocal
+from pants.contrib.node.tasks.node_task import NodeTask
+
+
+class NodeInstall(NodeTask):
+  """Installs a node_module target into the source directory
+
+  Note:
+    Running the node install on an example_project will install into the local source dir
+    rather than in the typical .pants.d working directory.
+
+    This task is intended to set up the environment for development purposes rather than
+    to run tests or other isolated tasks.
+
+  Example:
+    ./pants node-install src/node/example_project:example_project
+
+    This will produce a node_modules dir in `src/node/example_project/node_modules`
+  """
+
+  @classmethod
+  def prepare(cls, options, round_manager):
+    super(NodeInstall, cls).prepare(options, round_manager)
+    round_manager.require_data(NodePathsLocal)
+
+  @classmethod
+  def supports_passthru_args(cls):
```

Comment:
This doesn't seem to use `passthru_args`... does one of the parent classes? That would look like `./pants node-install $target -- $passthru_args`

Reply:
This is still relevant.